pandas/core/arrays/interval.py

```diff
@@ -727,7 +730,7 @@ def __getitem__(
         if np.ndim(left) > 1:
             # GH#30588 multi-dimensional indexer disallowed
             raise ValueError("multi-dimensional indexing not allowed")
-        return self._shallow_copy(left, right)
+        return self._shallow_copy(left, right, verify_integrity=False)
```

Comment:
couldnt we also skip _ensure_simple_new_inputs, i.e. just do

`return self._simple_new(left, right, dtype=self.dtype)`

Reply:
let's see if it works, probably not necessary